ui/src/main/java/com/epam/eco/kafkamanager/ui/topics/TopicController.java

```diff
@@ -91,14 +91,13 @@ public class TopicController {
     public static final String MAPPING_TOPICS = "/topics";
     public static final String MAPPING_TOPIC = MAPPING_TOPICS + "/{name}";
     public static final String MAPPING_COUNT_RECORDS = MAPPING_TOPIC + "/count_records";
-    public static final String MAPPING_EXPORT = "/topics_export";
     public static final String MAPPING_PURGER = MAPPING_TOPIC + "/purger";
     public static final String MAPPING_DELETE = MAPPING_TOPIC + "/delete";
-    public static final String MAPPING_CREATE = "/topic_create";
     public static final String MAPPING_CONFIG = MAPPING_TOPIC + "/config";
     public static final String MAPPING_PARTITIONS = MAPPING_TOPIC + "/partitions";
     public static final String MAPPING_METADATA = MAPPING_TOPIC + "/metadata";
-
+    public static final String MAPPING_EXPORT = "/topics_export";
```

Comment:
Sorry I can't understand what the changes are in this class. I see that some constants and methods have been moved in other places for some reason, and there are lots of formatting changes...